src/en/juju-backups.md

```diff
@@ -17,10 +17,11 @@ configuration files (such as '.jenv' files for running environments, or
 credentials) it is usually best to simply back up the entire Juju directory
 (`~/.juju`).
 
-!!! Note: On Windows systems, the Juju directory is in a different place
- (usually `C:\Users\<username>\AppData\Roaming\Juju`. Also, although `tar` is 
-available for Windows, you may prefer to use a more Windows-centric backup 
-application.
+!!! Note: 
+    On Windows systems, the Juju directory is in a different place
+     (usually `C:\Users\<username>\AppData\Roaming\Juju`. Also, although `tar` is 
```

Comment:
probably this

Reply:
I updated that paragraphs but the failure remains. Still looking.